.github/workflows/deploy.yaml

```diff
@@ -65,3 +65,36 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           context: .
+
+  monitor:
```

Comment:
This is based on the last time I did this in a node project. Unsure if this is 100% correct for a go project.